components/NoteEditor/Editor/Editor.js

```diff
@@ -0,0 +1,104 @@
+import { useState } from 'react';
+import dynamic from 'next/dynamic';
+
+import shortid from 'shortid';
+import styles from './Editor.module.scss';
+import Heading from './Heading';
+
+const Section = dynamic(() => import('./Section'), { ssr: false });
+
+function formatDate(dateObj) {
+  return new Intl.DateTimeFormat('en-US', { month: 'short', day: 'numeric' }).format(dateObj);
+}
+
+const formatedDate = formatDate(new Date()).split(' ');
+
+export default function Editor() {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [sections, setSections] = useState([{ id: shortid.generate(), keyword: '', content: '' }]);
```

Comment:
You should probably create the initial state outside of the Component so it doesnt get recreated on rerender